datasets/nli_tr/nli_tr.py

```diff
@@ -0,0 +1,167 @@
+# coding=utf-8
+# Copyright 2020 The HuggingFace Datasets Authors and the current dataset script contributor.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""NLI-TR: The Turkish translation of SNLI and MultiNLI datasets using Amazon Translate."""
+
+from __future__ import absolute_import, division, print_function
+
+import codecs
+import json
+import os
+
+import datasets
+
+
+_CITATION = """\
+@inproceedings{budur-etal-2020-data,
+    title = "Data and Representation for Turkish Natural Language Inference",
+    author = "Budur, Emrah and
+      \"{O}zçelik, Rıza and
+      G\"{u}ng\"{o}r, Tunga",
+    booktitle = "Proceedings of the 2020 Conference on Empirical Methods in Natural Language Processing (EMNLP)",
+    month = nov,
+    year = "2020",
+    address = "Online",
+    publisher = "Association for Computational Linguistics",
+    abstract = "Large annotated datasets in NLP are overwhelmingly in English. This is an obstacle to progress in other languages. Unfortunately, obtaining new annotated resources for each task in each language would be prohibitively expensive. At the same time, commercial machine translation systems are now robust. Can we leverage these systems to translate English-language datasets automatically? In this paper, we offer a positive response for natural language inference (NLI) in Turkish. We translated two large English NLI datasets into Turkish and had a team of experts validate their translation quality and fidelity to the original labels. Using these datasets, we address core issues of representation for Turkish NLI. We find that in-language embeddings are essential and that morphological parsing can be avoided where the training set is large. Finally, we show that models trained on our machine-translated datasets are successful on human-translated evaluation sets. We share all code, models, and data publicly.",
+}
+"""
+
+_DESCRIPTION = """\
+The Natural Language Inference in Turkish (NLI-TR) is a set of two large scale datasets that were obtained by translating the foundational NLI corpora (SNLI and MNLI) using Amazon Translate.
+"""
+
+_HOMEPAGE = "https://github.com/boun-tabi/NLI-TR"
+
+
+class NLITRConfig(datasets.BuilderConfig):
+    """ BuilderConfig for NLI-TR"""
+
+    def __init__(self, version=None, data_url=None, data_dir=None, **kwargs):

```

Comment:
```suggestion
    def __init__(self, version=None, data_url=None, **kwargs):
```
data_dir is not used here